doc/admin-guide/plugins/traffic_dump.en.rst

```diff
@@ -46,6 +46,10 @@ Plugin Configuration
 
    (`required`) - specifies the max disk usage N bytes (approximate). Traffic Dump will stop capturing new sessions once disk usage exceeds this limit.
 
+   .. option:: --sensitive-fields <field1,field2,...,fieldn>
+
+   (`optional`) - a comma seperatated list of HTTP field names whose values are considered sensitive information. Traffic Dump will not dump the incoming field values for any of these fields but will instead dump a generic value for them of the same length as the original up to a maximum field length of 2048 bytes. If this option is not used, a default list of "Cookie,Set-Cookie" is used. Providing this option overwrites that default list with whatever values the user provides.
```

Comment:
Why is there a maximum on the field length?  Wouldn't you want to test with the same length that the origin or client is sending?

Reply:
Good feedback. I'm increasing it to 128 KB.